docs/section1/Rule1-4.md

```diff
@@ -0,0 +1,37 @@
+# Section 1 - Rule 1-4
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 1-4  
+**Rule Description:** The PCI shall be less than or equal to the PCIt when calculated in accordance with Standard 90.1 2019, Section 4.2.1.1  
+**Rule Assertion:** Options are Pass/Fail  
+**Appendix G Section:** None  
+**90.1 Section Reference:** 4.2.1.1  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** RMD
+
+**Applicability Checks:** None
```

Comment:
This does not account for renewables but that is not clear from the rule description or applicability.

Reply:
Do you mean it does not account for the limit on renewables? PCI does include renewables. My understanding is this rule is making sure the project complies without consideration for the limit on renewables, then if the renewables exceed the 5% limit Rule 1-5 becomes applicable.

Reply:
Okay, I see. That makes sense.